Components/ListSearchResults/Partials/Header/_style.scss

```diff
@@ -1,59 +1,87 @@
+$search-form-spacing: 6px;
+
 .header {
   &-title {
-    @include reset-content-margins;
+    text-align: center;
+
+    > * {
+      font-size: 2.5rem;
+      font-weight: 700;
+      line-height: 1.25;
+      padding-top: 0;
+
+      @media (min-width: $breakpoint-mobile-horizontal) {
+        margin-bottom: 0.8em;
+      }
 
-    padding: $container-padding-desktop 0;
+      @media (min-width: $breakpoint-tablet) {
+        margin-bottom: 1.75em;
+      }
+    }
```

Comment:
This doesn't look like what I expected. Isn't it just a wysiwyg, so you could just add an h1 or h2 and would look like intended, without any additional styling (like it was before)?

Reply:
Was no option to change to heading from the admin panel on my local machine. 
Will remove this one.
![srch](https://user-images.githubusercontent.com/49150551/61356041-ccac6180-a875-11e9-99af-9254e9b4dc89.png)

Reply:
Please click the switch to "Visual" on the top right and you'll have the options.